<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>functionSetDlg</class>
 <widget class="BaseDlg" name="functionSetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>1167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">functionSetDlg</string>
  </property>
  <widget class="QWidget" name="box_widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>543</width>
     <height>1061</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QCheckBox" name="clot_sample_stop">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>加样停</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="reagent_r1">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>R1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="reagent_r2">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>R2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ultrasonic_r1">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>超声混匀R1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ultrasonic_r2">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>超声混匀R2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="incubation_stop">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>加样停/停机</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bubbles_acidity">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>酸液气泡传感器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bubbles_acidity_back">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>酸液备用气泡传感器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bubbles_alkalinity">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>碱液气泡传感器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bubbles_alkalinity_back">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>碱液备用气泡传感器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bubbles_sample_probe_s">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>样本针S气泡传感器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bubbles_reagent_probe_r1">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>试剂针R1气泡传感器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bubbles_reagent_probe_r2">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>试剂针R2气泡传感器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="time_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>170</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>170</height>
       </size>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>休眠时间（分钟）</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>156</x>
         <y>120</y>
         <width>10</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>不休眠时间限</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="lower_time_edit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="upper_time_edit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>120</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
       <property name="time">
        <time>
         <hour>12</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="btn_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>112</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>112</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>14</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>47</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>267</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_btn">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_btn">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
         <property name="buttons" stdset="0">
          <string notr="true">btn_dlg</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>functionSetDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>754</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>855</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
